portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,20 +45,30 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int commentLimit = Integer.parseInt(request.getParameter("limit"));
     String languageCode = request.getParameter("languageCode");
-    System.out.println(languageCode);
 
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     List<Entity> queryList = results.asList(FetchOptions.Builder.withLimit(commentLimit));
 
+    Translate translate = null;
+    if (!languageCode.equals("original")) {
+      translate = TranslateOptions.getDefaultInstance().getService();
+    }
```

Comment:
You could directly use
"Translate translate = TranslateOptions.getDefaultInstance().getService();"

You will overwrite the value anyway later when the language code is not original.